### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SmartSergiZ**
[DEMO LINK](https://SmartSergiZ.github.io/layout_calendar/)
[TEST REPORT LINK](https://SmartSergiZ.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,66 @@
...
+$month-length: 28, 29, 30, 31;
+
+html {
+  font-size: 30px;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  margin: 0 auto;
```
Comment (BudnikOleksii) [inline src/styles/main.scss]: BEM-block shouldn't have any margins, it you want to specify them use BEM mix instead.

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid black;
+    width: $ceil-width;
+    height: $ceit-height;
+    font-size: 30px;
+    transition: all 0.5s;
```
Comment (BudnikOleksii) [inline src/styles/main.scss]: It's better to specify what properties should apply the transition.
image

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Great work!
Left a few comments, check them.
Also, let's move variables to separate file.
And in accordance with task add cursor pointer:
image

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+  &__cell {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid black;
+    width: $ceil-width;
+    height: $ceit-height;
+    font-size: 30px;
```
Comment (alexander-ignatow) [inline src/styles/main.scss]: You already have `font-size` in the body - why duplicate it here?

In addition: per task:
> Add a number (Arial 30px)

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+    transition: transform 0.5s;
+
+    @for $i from 1 through 29 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
```
Comment (alexander-ignatow) [inline src/styles/main.scss]: The `cursor:pointer` is ok  👍 , but why are you setting it on hover?

**Review by alexander-ignatow — CHANGES_REQUESTED** [review]
Good job fixing comments - still some are left.

Let's fix them!

**Review by roman-mirzoian — CHANGES_REQUESTED** [review]
Almost done, but now the number of days doesn't change after updating the days modifier and the cursor doesn't change when hovering over a cell.
https://gyazo.com/0d8db61bf63b733b629be87e526b5167
image

**Review by roman-mirzoian — APPROVED** [review]
Well done 🔥

**Pull request closed without merging**